src/main/scala/edu/ie3/simona/model/grid/Transformer3wModel.scala

```diff
@@ -319,66 +323,56 @@ case object Transformer3wModel {
     val transformerRefSystem =
       RefSystem(transformerType.getsRatedA, transformerType.getvRatedA)
 
-    /* Extract the equivalent circuit diagram parameters from type, with the perspective of the
-     * transformer */
+    /* Get the physical equivalent circuit diagram parameters from type. They come with reference to the highest
+     * voltage side, therefore, in power flow case B and C, they need to be adapted. */
     val (rTrafo, xTrafo, gTrafo, bTrafo) = powerFlowCase match {
       case PowerFlowCaseA =>
         (
-          transformerRefSystem.rInPu(transformerType.getrScA),
-          transformerRefSystem.xInPu(transformerType.getxScA),
-          transformerRefSystem.gInPu(transformerType.getgM),
-          transformerRefSystem.gInPu(transformerType.getbM)
+          transformerType.getrScA,
+          transformerType.getxScA,
+          transformerType.getgM,
+          transformerType.getbM
         )
       case PowerFlowCaseB =>
+        val nominalRatio = transformerType
+          .getvRatedA()
+          .divide(transformerType.getvRatedB())
+          .asType(classOf[Dimensionless])
+          .to(AbstractUnit.ONE)
+          .getValue
+          .doubleValue()
         (
-          transformerRefSystem.rInPu(transformerType.getrScB),
-          transformerRefSystem.xInPu(transformerType.getxScB),
-          Quantities.getQuantity(0d, PU),
-          Quantities.getQuantity(0d, PU)
+          transformerType.getrScB.divide(pow(nominalRatio, 2)),
+          transformerType.getxScB.divide(pow(nominalRatio, 2)),
+          Quantities.getQuantity(0d, StandardUnits.ADMITTANCE),
+          Quantities.getQuantity(0d, StandardUnits.ADMITTANCE)
```

Comment:
with [PSDM#573](https://github.com/ie3-institute/PowerSystemDataModel/pull/573) we introduced Resistance, Reactance in addition to Impedance, as well as Conductance and Susceptance for Admittance. Does it make sense to use it here as well?